README.md

```diff
@@ -0,0 +1,34 @@
+
+# DPM360
+Repository for Disease Progression Modeling workbench 360
+
+Overview and YouTube demonstration are available [here](https://ibm.github.io/DPM360/). License, Contribution, Publications are also available there.
+
+<figure><center><img src=./docs/resources/png/dpm360v2.png "DPM360" width="300"/></center><figcaption>DPM360 Component View</figcaption></figure>
+
+# Installation Guides
+
+DPM360 components are interoperable but also work as independent tools. They are working on a cloud cluster (DPM360 server side) and on a client server (DPM360 client side), having separate installation procedures. Please see the guides below to install each component.
```

Comment:
suggested changes: 

**Change 1**
`also works as` -> `can also work as`

**Change 2**:
`They are working..` -> 
`DPM360 is typically installed over a cluster that sets up a number of interconnected micro-services. We can broadly divide the components into two groups viz (i) components that are concerned with micro service setups and (ii) standalone python packages providing core functional capabilities`